driver-tests/shading/shaded/src/test/java/com/datastax/driver/core/NettyUtilTest.java

```diff
@@ -0,0 +1,29 @@
+package com.datastax.driver.core;
+
+/*
```

Comment:
It seems like this passes our license check, but can you please move the license headers to above the package declaration to be consistent with other source files?